contracts/ExampleWithTreasury/PokeMeReady.sol

```diff
@@ -4,12 +4,27 @@ pragma solidity 0.8.0;
 abstract contract PokeMeReady {
     address payable public immutable pokeMe;
 
-    constructor(address payable _pokeMe) {
-        pokeMe = _pokeMe;
-    }
-
     modifier onlyPokeMe() {
         require(msg.sender == pokeMe, "PokeMeReady: onlyPokeMe");
         _;
     }
+
+    modifier onlyTaskCreator(address _taskCreator) {
```

Comment:
not sure if i understand this pice of code fully, but looking at Counter.sol, `_taskCreator` would allways be the contract deployer? Is the desired functionality?